lib/cell/concept.rb

```diff
@@ -5,8 +5,23 @@ class Concept < Cell::ViewModel
 
     # TODO: this should be in Helper or something. this should be the only entry point from controller/view.
     class << self
-      def class_from_cell_name(name)
-        name.camelize.constantize
+      def class_from_cell_name(name, controller)
+        @cell_class ||= begin
+          supports_path = controller.supports_path?
+          routes  = controller.respond_to?(:_routes)  && controller._routes
+          helpers = controller.respond_to?(:_helpers) && controller._helpers
+
+          Class.new(name.camelize.constantize) do
```

Comment:
The problem here is that we *had* all Rails helpers included in earlier Cells versions, and, believe it or not, but things broke everywhere and *I* had to fix it. Since Cells 4.0, my life has gotten better because now people include what they need and find out how weird Rails helpers are on their own. :stuck_out_tongue_winking_eye: